src/parse/mark.js

```diff
@@ -23,6 +36,17 @@ function parseMark(model, mark) {
   return mark;
 }
 
+// set color given in graph if "default" specified in spec for marks color
+function defaultColor(property, configProperty, prop, config) {
+  if (property in prop && 'value' in prop[property] && prop[property]['value'] === 'default') {
+    if (typeof config !== 'undefined' && 'marks' in config && configProperty in config.marks) {
+      prop[property]['value'] = config.marks[configProperty];
+    } else {
+      prop[property]['value'] = '#000000';
+    }
+  }
+}
+
```

Comment:
Yeah, let's go with `config.marks.fill`. That keeps it consistent with the property name it provides a default for (`fill`) if the user hasn't explicitly provided one. Thanks!